src/pages/Contributing/DesignTokens/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: 'Design Tokens'
+order: 13
+---
+
+## Definition
+
+Design tokens are named entities that store visual design information. These are used in place of hard-coded values (such as hex values for color or pixels for spacing) in order to maintain a scalable, consistent system for UI development.
+
+## How does it works ?
+
+Design tokens are defined in json files that are transpiled to multiple formats (scss, js, xml etc...) to be used on various platforms
+For example, with css, the tokens are transpiled from JSON to SCSS variables as well as SCSS maps.
+
+We use [style-dictionary](https://github.com/amzn/style-dictionary), a solution developped by Amazon, to manage them.
+
+We can define any transpilation template/format we need to acomodate our needs.
+
+## Why using design tokens ?
+
+By compiling those tokens in multiple formats, we are able to keep things like colors in sinc on all platforms and tech.
+Changing a color in the source file will change css, ios, android and javascript based project at the same time.
+
+## We use two "layers" of tokens
+
+1 - the base, low level layer
+
+To define all the basic values like colors, sizes etc.
+As an example, we define all the possible colors in the same place.
+
+Most of the time, you should not have to use thoses variables directly, because a highter level of variable or a mixin should be available to provide authorized values for the type of property/element you are trying to define.
+
+2 - an "elements" level layer
+
+For example, we create a list of authorized values for text-colors or create some tokens to define buttons cosmetic styles. This layer make reference to the base layers values.
+
+In the documentation you may read them expressed as js object keys selectors : color.button.solid.background when compiled to SCSS the dots are replaced with simple dashes - the scss variable will be \$color-button-solid-background.
```

Comment:
To improve lisibility : 
- color.button.solid.background => `color.button.solid.background`
- \$color-button-solid-background => `$color-button-solid-background`